images/hugo/README.md

```diff
@@ -42,25 +35,53 @@ hugo v0.119.0-b84644c008e0dc2c4b67bd69cccf87a41a03937e linux/amd64 BuildDate=202
 
 ## Application Setup for End Users
 
-Here is an example using the Hugo image to run the
-["quickstart"](https://gohugo.io/getting-started/quick-start/#commands) locally:
+The following is an example of using the Hugo image locally. It's based on the official [Hugo "quickstart"](https://gohugo.io/getting-started/quick-start/#commands) example.
```

Comment:
It annoys me so much that they call it "quick start" in the documentation but you run it as `quickstart`